README.md

```diff
@@ -214,6 +214,7 @@ Use `UTC`, `Local` or pick a timezone name from the [(IANA) tz database](https:/
 | `--excluded-times-of-day` | times of day when chaos is to be suspended, e.g. "22:00-08:00"       | (no times of day excluded) |
 | `--excluded-days-of-year` | days of a year when chaos is to be suspended, e.g. "Apr1,Dec24"      | (no days of year excluded) |
 | `--timezone`              | timezone from tz database, e.g. "America/New_York", "UTC" or "Local" | (UTC)                      |
+| `--max-runtime`           | Maximum runtime before chaoskube exits                               | -1s (infinite time)        |
```

Comment:
I would also be fine to use `0` as the pivot element, whatever you prefer.

Reply:
we can leave it as -1s as the default value